src/appservice-manager.ts

```diff
@@ -34,16 +34,16 @@ export class AppserviceManager extends Manager {
 
   constructor () {
     super()
-    log.verbose('AppserviceManager', 'constructor()')
+    log.verbose('Appservicemanager.js', 'constructor()')
```

Comment:
We should keep the module name as the `AppserviceManager` in the log messages.

Reply:
Okay, I was too careless when using global substitution, so sorry for that. Why you and Jiarui can find such detailed questions, it is amazing to me.

Reply:
No problem.

We just pay more attention to those details.